packages/multichain-network-controller/src/utils.test.ts

```diff
@@ -32,4 +35,69 @@ describe('utils', () => {
       expect(checkIfSupportedCaipChainId('eip155:1')).toBe(false);
     });
   });
+
+  describe('updateNetworkConfiguration', () => {
+    it('updates the network configuration a single EVM network', () => {
+      const network: NetworkConfiguration = {
+        chainId: '0x1',
+        name: 'Ethereum Mainnet',
+        nativeCurrency: 'ETH',
```

Comment:
There was a comment about the ticker in the network configuration not being able to be fully translate to Caip for EVM since we don't follow a standard

Reply:
Currently, the `CommonNetworkConfiguration` is using `CaipAssetType` but on the ADR, the `nativeCurrency` is just a plain `string`.

IMO we should force use a `CaipAssetType` (we should maybe fix this on the ADR?), but we would need a way to map any ticker to their CAIP-19 equivalent...

The main reason I would push for CAIP-19 here, is because ALL new multichain controllers uses CAIP everywhere, so using the ticker here would just delay the problem I stated above (about the mapping ticker <> CAIP-19). It's better to fix it on the configuration.

WDYT?

Reply:
I understand and agree with that point, but I the problem I see is that this value is purely used for UI purposes. So the two options can be seen as,

1. `nativeCurrency` is a `string`: UI gets the value and directly uses it
2. `nativeCurrency` is a `CaipAssetType`: UI gets the value, tries to translate to ticker value to be shown in the UI (possible unhandled cases). If the ticker value is obtained, display it in the UI, if not display "fallback"

Reply:
Based on our conversation. https://github.com/MetaMask/core/pull/5315/commits/11d4258c4f8be688bfc3d54757b4735ebf6b97b2

Reply:
As discussed internally, now that both configurations have different types, we're not forced to use CAIP-19 for EVM. So we can use the union-tagged with `isEvm` to use one or the other (and the type will be enforced as well).

```ts
const ticker = config.isEvm
  ? config.nativeCurrency
  : toTicker(config.nativeCurrency /* CAIP-19 */);